pkg/apis/contrail/v1alpha1/manager_types.go

```diff
@@ -37,6 +37,7 @@ type Services struct {
 	Keystone         *Keystone         `json:"keystone,omitempty"`
 	Swift            *Swift            `json:"swift,omitempty"`
 	Memcached        *Memcached        `json:"memcached,omitempty"`
+	ContrailCNI      []*ContrailCNI    `json:"contrailCNI,omitempty"`
```

Comment:
Done.
Array in case we'd like different configuration for nodes with different node selector - what if some nodes were supposed to connect to one controller and some to another?

Reply:
If we introduce that kind of functionality we should also cover it by tests.